docs/changelog.md

```diff
@@ -29,10 +32,6 @@ We use _breaking :warning:_ to mark changes that are not backward compatible \(r
 * [\#340](https://github.com/tellor-io/telliot/pull/340) Manifest files to run in k8s google cloud with Prometheus and Grafana monitoring. The team will run a public telliot miner dashboard at [http://monitor.tellor.io/](http://monitor.tellor.io/)
 * [\#334](https://github.com/tellor-io/telliot/pull/334) DEFITVL feed as a new request ID 57. Miners would have to update the binary and `index.json` to be able to submit.
 
-### Fixed
-
-* [\#359](https://github.com/tellor-io/telliot/pull/359) Don't resubmit when the contract returns an error - "Miner already submitted the value".
-
```

Comment:
This is an overlook, as the it shouldn't be in the changelog yet. The PR is still WIP.